release/src/main/scripts/build_release_candidate.sh

```diff
@@ -215,6 +215,7 @@ if [[ $confirmation = "y" ]]; then
   git checkout ${RELEASE_BRANCH}
   cd sdks/python && tox -e docs
   GENERATED_PYDOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_PYTHON_DOC}/${BEAM_ROOT_DIR}/sdks/python/target/docs/_build
+  rm -rf ${GENERATED_PYDOC}/.doctrees
```

Comment:
Another option would be to add this `.doctrees` directory to the `.gitignore` file; then it is less likely to be accidentally included if a release manager does this process manually.

Reply:
Added to .gitignore too.